### PR TITLE
Added helper function for VM Backup

### PR DESCRIPTION
**Pull request by mkoppal-px**
**What this PR does / why we need it**:
Added helper function for VM Backup. Needed for Px-Backup Kubevirt Phase-2 2.7.0 release automation.

**Which issue(s) this PR fixes** (optional)
Closes #PB-5790

**Special notes for your reviewer**:
Cannot test since no build is available

### CODE REVIEW

**Review by atrivedi-px — APPROVED** [review]
LGTM

**Review by ak-px — APPROVED** [review]
LGTM

**Review by aravindranath14 — APPROVED** [review]
LGTM

File: tests/backup_helper.go [hunk 1]
```diff
@@ -321,6 +321,23 @@ func CreateBackup(backupName string, clusterName string, bLocation string, bLoca
 	return nil
 }
 
+// CreateVMBackup creates a VM backup and checks for success
+func CreateVMBackup(backupName string, vms []kubevirtv1.VirtualMachine, clusterName string, bLocation string, bLocationUID string,
```
[the remark this answers is not in the document]
Comment (mkoppal-px) [reply tests/backup_helper.go:325]: Yes, user can give customer pre and post exec rule to be run

File: tests/backup_helper.go [hunk 2]
```diff
@@ -475,6 +492,35 @@ func CreateBackupWithValidation(ctx context1.Context, backupName string, cluster
 	return ValidateBackup(ctx, backupName, orgID, scheduledAppContextsToBackup, make([]string, 0))
 }
 
+func CreateVMBackupWithValidation(ctx context1.Context, backupName string, vms []kubevirtv1.VirtualMachine, clusterName string, bLocation string, bLocationUID string, scheduledAppContextsToBackup []*scheduler.Context, labelSelectors map[string]string, orgID string, uid string, preRuleName string, preRuleUid string, postRuleName string, postRuleUid string, skipVMAutoExecRules bool) error {
```
[the remark this answers is not in the document]
Comment (mkoppal-px) [reply tests/backup_helper.go:495]: Yes, user can give customer pre and post exec rule to be run

**Review by vpinisetti-px — APPROVED** [review]
LGTM

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Pull request merged**